include/aspect/particle/interpolator/bilinear_least_squares.h

```diff
@@ -54,6 +54,25 @@ namespace aspect
 
           // avoid -Woverloaded-virtual:
           using Interface<dim>::properties_at_points;
+
+          /**
+           * Declare the parameters this class takes through input files.
+           */
+          static
+          void
+          declare_parameters (ParameterHandler &prm);
+
+          /**
+           * Read the parameters this class declares from the parameter file.
+           */
+          virtual
+          void
+          parse_parameters (ParameterHandler &prm);
+
+        private:
+          bool use_global_valued_limter;
```

Comment:
I am calling this a limiter, although a simplified version, because this is in the same spirit of the parameters

```
Use limiter for discontinuous composition solution
Global temperature maximum
Global temperature minimum
```
which are part of the Discretization subsection. In the near future, we plan on reusing the already implemented limiting scheme for temperature and composition.